recipes/texture-atlas/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: texture-atlas
+  version: {{ environ['GIT_DESCRIBE_TAG'] ~ "." ~ environ['GIT_DESCRIBE_NUMBER'] ~ "." ~ environ['GIT_DESCRIBE_HASH'] }}
+
+source:
+#  git_rev: {{ sha }}
+  git_branch: add-findproj4
+  git_url: git@gitlab.kitware.com:danlipsa/core3d-keu.git
```

Comment:
I don't have access to this repository. Can you add the required changes to the main repo?

Reply:
Yes, I have to merge my changes to the keu code first.